test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I like the idea of grouping all the V2 CPr tests in one directory, `0787_cpr_full_functional_v2` for example.
And removing the `_in_ngsiv2` in those from 0787 plus moving all tests files from 2193 to that new directory.